src/main/java/io/confluent/kafkarest/SimpleConsumerManager.java

```diff
@@ -210,9 +213,10 @@ private JsonConsumerRecord createJsonConsumerRecord(final MessageAndOffset messa
                                                       final String topicName,
                                                       final int partitionId) {
     final MessageAndMetadata<Object, Object> messageAndMetadata =
-        new MessageAndMetadata<Object, Object>(topicName, partitionId,
-            messageAndOffset.message(), messageAndOffset.offset(),
-            jsonDecoder, jsonDecoder);
+        new MessageAndMetadata<>(topicName, partitionId,
+                                 messageAndOffset.message(), messageAndOffset.offset(),
+                                 0, TimestampType.CREATE_TIME,
```

Comment:
Tbh, these extra timestamp parameters have no impact on the existing tests since they are completely new. The exact values didn't matter.